features/step_definitions/_constants.rb

```diff
@@ -1,11 +1,11 @@
 OPT_SYM = '(?: :(\S+))?'
 SYM = ':(\S+)'
-OPT_QUOTED = '(?: "(.+?)")?'
-QUOTED = '"(.+?)"'
+OPT_QUOTED = '(?: "([^"\n]+)")?'
+QUOTED = '"([^"\n]+)"'
 HTTP_URL = '(https?:\/\/.+)'
 USER = 'the( \\S+)? user'
-NUMBER = '([0-9]+|<%=.+?%>)'
-WORD = '(\w+|<%=.+?%>)'
+NUMBER = '([0-9]+|<%=(?!%>).+?%>)'
+WORD = '(\w+|<%=(?!%>).+?%>)'
 OPT_WORD = "(?: #{WORD})?"
 # use for regular expression
-RE='/(.+?)/'
+RE='/([^/\n]+)/'
```

Comment:
we should allow `/` in regular expressions

Reply:
I'm not sure how, given that `/` is the delimiter.  The non-greedy `.+?` will stop at the first `/`. 

```ruby
>> "/asdf/".match('/([^/\n]+)/')
=> #<MatchData "/asdf/" 1:"asdf">
>> "/as/df/".match('/([^/\n]+)/')
=> #<MatchData "/as/" 1:"as">
>> "/asdf/".match('/(.+?)/')
=> #<MatchData "/asdf/" 1:"asdf">
>> "/as/df/".match('/(.+?)/')
=> #<MatchData "/as/" 1:"as">
```

Reply:
It will try that, but then the rest of the step will likely not match, so there will be a backtrack where the `/` is used as part of the RE. 
```
[13] pry(main)> 'asd /first re with/ slach/ and /second re with / slash/ yo' =~ /asd \/(.+?)\/ and \/(.+?)\/ yo/
=> 0
```